test/e2e/openshift/openshift_test.go

```diff
@@ -85,6 +86,44 @@ var _ = Describe("Azure Container Cluster using the OpenShift Orchestrator", fun
 		}
 	})
 
+	It("should deploy all pods onto appropriate nodes", func() {
+		namespaces := []string{
+			"openshift-metrics",
+			"e2e-test",
+		}
+
+		daemonsetPrefixes := []string{
```

Comment:
ideally you would get rid of daemonsetPrefixes entirely; use "pod has an OwnerReference of kind DaemonSet" to detect a pod owned by a daemonset.